mmv1/third_party/terraform/services/gemini/resource_gemini_code_repository_index_test.go.tmpl

```diff
@@ -0,0 +1,73 @@
+package gemini_test
+{{- if ne $.TargetVersionName "ga" }}
+
+import (
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/plancheck"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+)
+
+func TestAccGeminiCodeRepositoryIndex_update(t *testing.T) {
+	context := map[string]interface{}{
+		"project_id": os.Getenv("GOOGLE_PROJECT"),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderBetaFactories(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccGeminiCodeRepositoryIndex_basic(context),
+			},
+			{
+				ResourceName:            "google_gemini_code_repository_index.example",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"code_repository_index_id", "labels", "location", "terraform_labels"},
+			},
+			{
+				Config: testAccGeminiCodeRepositoryIndex_update(context),
+				ConfigPlanChecks: resource.ConfigPlanChecks{
+					PreApply: []plancheck.PlanCheck{
+						plancheck.ExpectResourceAction("google_gemini_code_repository_index.example", plancheck.ResourceActionUpdate),
+					},
+				},
+			},
+			{
+				ResourceName:            "google_gemini_code_repository_index.example",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"code_repository_index_id", "labels", "location", "terraform_labels"},
+			},
+		},
+	})
+}
+
+func testAccGeminiCodeRepositoryIndex_basic(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_gemini_code_repository_index" "example" {
+  provider = google-beta
+  labels = {"ccfe_debug_note": "gcloud_e2e_do_not_delete"}
```

Comment:
IIUC this CRI is re-created. `gcloud_e2e_do_not_delete` looks incorrect and if I'm right, it should be rather `terraform_e2e_should_be_deleted` or something like that. Please, check other occurrences of the label definition